sdk/identity/azure-identity/azure/identity/_credentials/certificate.py

```diff
@@ -45,10 +45,22 @@ class CertificateCredential(ClientCredentialBase):
     :keyword cache_persistence_options: configuration for persistent token caching. If unspecified, the credential
         will cache tokens in memory.
     :paramtype cache_persistence_options: ~azure.identity.TokenCachePersistenceOptions
+    :keyword bool allow_broker:  
+        Brokers provide Single-Sign-On, device identification,
+        and application identification verification.
+        If this parameter is set to True,
+        the broker will be used when possible. Default to False.
```

Comment:
Are the line breaks here intentional? The following has the same line breaking as the rest of the docstring:
```suggestion
    :keyword bool allow_broker: Brokers provide single sign-on, device identification, and application identification
        verification. If this parameter is set to True, the broker will be used when possible. Defaults to False.
```